### PR TITLE
Currency update

### PR DESCRIPTION
**Pull request by 3rdvision**
| - | - |
| --- | --- |
| Issue | - |
| Decisions |  Adds more currency symbols according to ISO standard currency codes (https://apps.cybersource.com/library/documentation/sbc/quickref/currencies.pdf); Fix value type to only accept numbers; Fix zero places - when a currency has zero places `return currencyInfo.places || fallback;` would always return `fallback` instead of `currencyInfo.places` holding the value 0. |
| Animated GIF |   |

### CODE REVIEW

File: vue/components/ui/molecules/input-currency/input-currency.vue [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const InputCurrency = {
         },
         _places(currency, fallback = 2) {
             const currencyInfo = currencies[currency] || {};
-            return currencyInfo.places || fallback;
+            return "places" in currencyInfo ? currencyInfo.places : fallback;
```
Comment (gcandal) [inline vue/components/ui/molecules/input-currency/input-currency.vue]: Your approach is sound, but please change to be similar with the way we usually do this kind of stuff: `currencyInfo.places === undefined ? fallback : currencyInfo.places`

File: vue/components/ui/molecules/input-currency/input-currency.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const InputCurrency = {
             default: () => ({})
         },
         value: {
-            type: String,
+            type: Number,
```
Comment (gcandal) [inline vue/components/ui/molecules/input-currency/input-currency.vue]: Maybe we should accept both `String` and `Number`. Then, we can convert it to a `Number` every time we copy this to `valueData`.

File: vue/components/ui/molecules/input-currency/input-currency.stories.js [hunk 3]
```diff
@@ -8,15 +8,27 @@ storiesOf("Molecules", module)
     .add("Input Currency", () => ({
         props: {
             value: {
-                default: text("Value", "50")
+                default: text("Value", 50)
```
Comment (gcandal) [inline vue/components/ui/molecules/input-currency/input-currency.stories.js]: `number`

File: vue/components/ui/molecules/input-currency/input-currency.vue [hunk 4]
```diff
@@ -28,7 +28,7 @@ export const InputCurrency = {
             default: () => ({})
         },
         value: {
-            type: String,
+            type: String | Number,
```
Comment (joamag) [inline vue/components/ui/molecules/input-currency/input-currency.vue]: why the or?

Comment (gcandal) [reply vue/components/ui/molecules/input-currency/input-currency.vue]: To allow both types, similarly to what we already do in `input`

**Pull request merged**